poster/index.html

```diff
@@ -15,64 +15,71 @@
         16. Juli - 19. Juli
     </header>
 
-    <figure class="texture">
-        <img src="https://www.dropbox.com/s/3a7tybohjrtaz72/gradient_triangle.png?raw=1" alt="noise-patterns">
-    </figure>
-
-    <section class="primary-details u-text_main">
-        <p> Ausstellung der Studiengänge Kommunikations - design </p>
-        <p> Öffnungszeiten der Ausstellung </p>
-        <ul class="remove-list">
-            <li> 17. Juli 12:00-18:00 </li>
-            <li> 18. Juli 10:00-17:00 </li>
-            <li> 19. Juli 10:00-17:00 </li>
-        </ul>
+    <div class="triangle-pattern"></div>
+
+    <section class="primary-details_flex">
+        <div class="flex-grid u-text_details primary-details">
+            <p> Ausstellung der Studiengänge Kommunikations - design </p>
+            <p> Öffnungszeiten der Ausstellung </p>
+            <ul class="u-remove-list">
+                <li> 17. Juli 12:00-18:00 </li>
+                <li> 18. Juli 10:00-17:00 </li>
+                <li> 19. Juli 10:00-17:00 </li>
+            </ul>
+        </div>
+        <div class="noise_pattern flex-grid">
+        </div>
     </section>
 
-    <figure class="texture">
-        <img src="https://www.dropbox.com/s/rfga556mflcpfp5/noise_pattern.png?raw=1" alt="noise-patterns" class="noise-pattern">
-        <!--   <img src= "https://www.dropbox.com/s/xe2evshywwz5dc7/horizontal_stripes.png?raw=1" alt= "noise-patterns" class= "horizontal-pattern" -->
-    </figure>
+    <div class="noise-horizontal_pattern"></div>
 
-    <section class="secondary-details u-text_main">
+    <section class="secondary-details u-text_details">
         <p> Vernissage und Abschlussfeier </p>
         <p> 16. juli </p>
         <p> 18:30 - 21:00 Uhr Eröffnung der Ausstellung </p>
         <p> Ab 22:00 Uhr ist Party. </p>
-        <p> Auftritt der Band >Fiste de pure emociòn< </p> <ul class="remove-list">
+        <p> Auftritt der Band >Fiste de pure emociòn< </p> <ul class="u-remove-list">
```

Comment:
FYI: Not deducting points for this, but a thing to be aware of! Here you're using the less than and greater than symbols as typographic elements, but that can be confusing to HTML, since those elements are used to wrap around elements, etc. This has the potential to create errors or breaks in your HTML.

It's easy to fix, however, because there's [special character codes](https://www.rapidtables.com/web/html/html-codes.html) to write those bits out:
`>` is `&gt;` (stands for "greater than")
and `<` is `&lt;` ("less than")

This will allow you to use those characters and not see HTML errors 🙌 
```suggestion
        <p> Auftritt der Band &gt;Fiste de pure emociòn&lt; </p> <ul class="u-remove-list">
```